src/cpufreq.c

```diff
@@ -89,11 +112,46 @@ int cpufreq_init_cluster(const struct cluster_t *cluster, const struct feat_t *f
     }
 
     /* Unknown */
-    write64(cluster->base + 0x440f8, 1);
+    if ((chip_id > T8015 && chip_id < S5L8960X) || chip_id < T7000)
```

Comment:
I'd really avoid all those > and < operations on chip_id and just hardcode a list where we need to do this write instead like e.g. the switch below